src/controller.rs

```diff
@@ -264,6 +264,57 @@ impl TnuaController {
     pub fn action_flow_status(&self) -> &TnuaActionFlowStatus {
         &self.action_flow_status
     }
+
+    /// Checks if the character is currently airborne based on the dynamic basis.
+    ///
+    /// Returns `Some(true)` if the character is airborne, `Some(false)` if the character is
+    /// grounded, and `None` if there is no dynamic basis available.
```

Comment:
This makes it sound like a basis (no need for the word "dynamic". In `dynamic_basis` it refers to how the basis is accessed, not to the nature of the basis) is something that maybe exists and maybe not, which is not really the case - without a basis Tnua will not work at all.

I'm starting to think that maybe this should return a `Result` with a unit error type that says there is no basis. But I can also do that after merging.